web/src/message_lists.ts

```diff
@@ -42,7 +42,17 @@ export let home: MessageList | undefined;
 export let current: MessageList | undefined;
 
 export function set_current(msg_list: MessageList): void {
+    if (msg_list !== home) {
+        home?.view.$list.removeClass("focused-message-list");
+    }
+
+    if (current !== home) {
+        // Remove old msg list from DOM.
+        current?.view.$list.remove();
+    }
+
     current = msg_list;
+    current.view.$list.addClass("focused-message-list");
 }
```

Comment:
This may be a problem for the call during initialization in `web/src/message_list.js`; we don't actually want to mark it focused there when we're actually about the load inbox or something.

Reply:
Makes sense, done.